### PR TITLE
Add common API docs

### PR DESCRIPTION
**Pull request by akegaviar**
Added common description API docs & API section

Closes https://github.com/chainstack/api/issues/6

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -178,6 +178,16 @@ module.exports = {
                     },
                 ]
             },
+            {
+                title: 'API',
+                collapsable: true,
+                children: [
+                    '/api/',
```
Comment (CaJIbHuK) [inline docs/.vuepress/config.js:185]: There may be a problem with such paths.
Currently, `https://docs.dev.chainstack.com/api/` redirects to the redoc.

Comment (CaJIbHuK) [reply docs/.vuepress/config.js:185]: So, we should rename either redoc path or this paths, so that they don't clash

Comment (CaJIbHuK) [reply docs/.vuepress/config.js:185]: @jona-sc wdyt?

File: docs/api/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    content: api blockchain node network manage
+---
+
+# Introduction
+
+Chainstack API allows you to manage:
+
+* your [organization](/glossary/organization);
+* your [projects](/glossary/project);
+* your [nodes](/glossary/node);
```
Comment (easeev) [inline docs/api/README.md:15]: Perhaps networks first?

File: docs/api/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* your [networks](/glossary/network).
+
+To start with the API, you must [create API keys](/api/create-api-keys).
+
+For all available API operations, see <a href="https://api.chainstack.com/" target="_blank">API reference</a>.
```
Comment (easeev) [inline docs/api/README.md:20]:
```suggestion
For all available API operations, see <a href="https://docs.chainstack.com/api/" target="_blank">API reference</a>.
```

Comment (easeev) [inline docs/api/README.md:20]: It's currently on https://docs.dev.chainstack.com/api/

Comment (akegaviar) [reply docs/api/README.md:20]: @CaJIbHuK is it not going to be on api.chainstack.com on prod?

File: docs/.vuepress/config.js (hunk 1 above)
Comment (easeev) [reply docs/.vuepress/config.js:185]: We can do `/api/reference` for API docs

File: docs/api/README.md (hunk 3 above)
Comment (CaJIbHuK) [reply docs/api/README.md:20]: `https://api.chainstack.com` it's a hosthame that should be used in API calls.
`https://docs.chainstack.com/api/` - it's a documentation of the API methods.

File: docs/.vuepress/config.js (hunk 1 above)
Comment (CaJIbHuK) [reply docs/.vuepress/config.js:185]: https://github.com/chainstack/helm-charts/compare/feature/change-apidoc-url-path-prefix?expand=1

File: docs/.vuepress/config.js [hunk 4]
```diff
@@ -178,6 +178,16 @@ module.exports = {
...
+                    '/api/create-api-keys',
+                    '/api/delete-api-keys',
+                    '/api/api-reference',
```
Comment (easeev) [inline docs/.vuepress/config.js:188]:
```suggestion
                    '/api/reference',
```

File: docs/api/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+Chainstack API allows you to manage:
+
+* your [organization](/glossary/organization);
+* your [projects](/glossary/project);
+* your [networks](/glossary/network);
+* your [nodes](/glossary/node).
+
+To start with the API, you must [create API keys](/api/create-api-keys).
+
+For all available API operations, see <a href="https://docs.chainstack.com/api/reference" target="_blank">API reference</a>.
```
Comment (easeev) [inline docs/api/README.md:20]: Why not

```suggestion
For all available API operations, see [API reference](/api/reference).
```

File: docs/api/api-reference.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+meta:
+  - name: description
+    content: Chainstack API reference to manage blockchain nodes, networks, projects, and organizations.
+  - name: keywords
+    content: api reference blockchain node network
+---
+
+# API reference
+
+See the API reference with examples at <a href="https://api.chainstack.com/" target="_blank">Chainstack API</a>.
```
Comment (easeev) [inline docs/api/api-reference.md:11]: 🤔

```suggestion
See the API reference with examples at [Chainstack API](/api/reference).
```

File: docs/api/README.md (hunk 5 above)
Comment (akegaviar) [reply docs/api/README.md:20]: My bad, it was coming from the `api.chainstack.com` and missed to fix the ref.

File: docs/api/create-api-keys.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    content: create api keys blockchain node network
+---
+
+# Create API keys
+
+You must create API keys to authenticate your requests to the Chainstack API.
+
+To create an API key:
+
+1. Go to the <a href="https://chainstack.com/user/settings/api-keys" target="_blank">API keys</a> section.
```
Comment (easeev) [inline docs/api/create-api-keys.md:15]: 🤔🤔

```suggestion
1. Go to the <a href="https://console.chainstack.com/user/settings/api-keys" target="_blank">API keys</a> section.
```

File: docs/api/create-api-keys.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+::: warning
+The API key value is only shown once and cannot be retrieved.
+
+Make sure you keep the API key secure.
+:::
+
+::: tip See also
+
+* [API reference](/api/api-reference)
```
Comment (easeev) [inline docs/api/create-api-keys.md:28]:
```suggestion
* [API reference](/api/reference)
```

File: docs/api/delete-api-keys.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    content: delete api keys blockchain node network
+---
+
+# Delete API keys
+
+You can delete your existing API keys.
+
+To delete an API key:
+
+1. Go to the <a href="https://chainstack.com/user/settings/api-keys" target="_blank">API keys</a> section.
```
Comment (easeev) [inline docs/api/delete-api-keys.md:15]:
```suggestion
1. Go to the <a href="https://console.chainstack.com/user/settings/api-keys" target="_blank">API keys</a> section.
```

File: docs/api/delete-api-keys.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+1. Click the edit icon next to the key you want to delete.
+1. Click **Delete**.
+
+::: tip See also
+
+* [Create API keys](/api/create-api-keys)
+* [API reference](/api/api-reference)
```
Comment (easeev) [inline docs/api/delete-api-keys.md:22]:
```suggestion
* [API reference](/api/reference)
```

File: docs/api/README.md (hunk 5 above)
Comment (easeev) [reply docs/api/README.md]: Answering my own question:

```
2 dead links was found

  File is not found: [API reference](/api/reference) (/home/circleci/project/docs/api/README.md:20:39)
  File is not found: [Chainstack API](/api/reference) (/home/circleci/project/docs/api/api-reference.md:11:40)
```

**Pull request merged**